### ISSUE TITLE
Add Laravel 9 support?

### ISSUE THREAD

**Issue by mikemartin** [aerni/laravel-sync#1]
Getting this error on a Laravel 9 installation:
```
aerni/sync v1.2.0 requires illuminate/support ^7.0 || ^8.0 -> found illuminate/support[v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
```

**Comment by aerni** [comment]
That's because the addon is not yet compatible with Laravel 9. Will try releasing an update soon.

**Comment by aerni** [comment]
Just released v1.3.0 that is compatible with Laravel 9.
